### PR TITLE
Wifi improvements

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by grekiki** [comment]
We now detect wrong passwrods and request the correct password from the user. We also write "CONNECTING" if we are actively connecting to a network. 

For a tiny usability upgrade, we now display "Refreshing networks" text, while loading the wifi networks available around us.

File: selfdrive/ui/qt/offroad/wifiManager.cc [hunk 1]
```diff
@@ -259,3 +295,12 @@ QString WifiManager::get_adapter(){
 
   return adapter_path;
 }
+
+void WifiManager::change(unsigned int a,unsigned int b,unsigned int c){
+  raw_adapter_state = a;
+  if(a==60 && c==8){
+    wrongPassword(connecting_to_network);
```
Comment (pd0wm) [inline selfdrive/ui/qt/offroad/wifiManager.cc:302]: use `emit` for signals

File: selfdrive/ui/qt/offroad/wifiManager.cc [hunk 2]
```diff
@@ -259,3 +295,12 @@ QString WifiManager::get_adapter(){
 
   return adapter_path;
 }
+
+void WifiManager::change(unsigned int a,unsigned int b,unsigned int c){
+  raw_adapter_state = a;
+  if(a==60 && c==8){
```
Comment (pd0wm) [inline selfdrive/ui/qt/offroad/wifiManager.cc:301]: Make defines for these magic values at the top of the file.

**Comment by pd0wm** [comment]
Remove all qDebugs()

**Pull request merged**